source/beamline/BioXAS/BioXASUtilities.h

```diff
@@ -69,7 +72,7 @@ class BioXASUtilities : public BioXASUtilitiesGroup
 	/// Returns the pressure monitors control.
 	BioXASUtilitiesGroup* pressureMonitors() const { return pressureMonitors_; }
 	/// Returns the temperature monitors control.
-	BioXASUtilitiesGroup* temperatureMonitors() const { return temperatureMonitors_; }
+	AMTemperatureMonitorGroup* temperatureMonitors() const { return temperatureMonitors_; }
```

Comment:
Since you changed the name of the class, should the getter should reflect that too?